Weekly_Work/Week2/Metrics_Root_class.py

```diff
@@ -0,0 +1,98 @@
+import json
+import datetime
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+class Metric():
+    
+    def __init__(self, path, source_code_exclude_list=None):
+
```

Comment:
Please include head comments at least for defining the arguments to the functions. It is difficult to follow otherwise.

Reply:
Will do!